src/julia.h

```diff
@@ -1105,6 +1105,7 @@ static inline int jl_is_layout_opaque(const jl_datatype_layout_t *l) JL_NOTSAFEP
 #define jl_is_svec(v)        jl_typeis(v,jl_simplevector_type)
 #define jl_is_simplevector(v) jl_is_svec(v)
 #define jl_is_datatype(v)    jl_typeis(v,jl_datatype_type)
+#define jl_is_function(v)    jl_typeis(v,jl_function_type)
```

Comment:
Ah, modulo this line --- did you mean to consume it later in the commit, perhaps in the `jl_function_type` branch?

Reply:
Oops, you're totally right. Ultimately i did not need this. Thanks! :)